modules/Sources/Features/NHHome/Transfer/TransferStore.swift

```diff
@@ -6,21 +6,91 @@
 //
 
 import ComposableArchitecture
+import Receive
+import NHSendFlow
+import TopUp
+import Utils
+import ZcashLightClientKit
+
+public typealias TransferStore = Store<TransferReducer.State, TransferReducer.Action>
+public typealias TransferViewStore = ViewStore<TransferReducer.State, TransferReducer.Action>
 
 public struct TransferReducer: ReducerProtocol {
-    public struct State: Equatable {}
+    let networkType: NetworkType
```

Comment:
Could be, probably. I'm not exactly a fan of how environments are configured in this project, I like your idea of using xcconfig files for the. I have avoided making significant changes like that as to limit the amount of merge conflicts, but I am beginning to think it may be time to consider nighthawk 2.0 diverged enough that it doesn't make sense to try and keep syncing upstream, unless it's some major fix. If we do that, we have the flexibility to make the codebase what we want it to be and not be limited by ECC codestyles, or odd architectural choices.